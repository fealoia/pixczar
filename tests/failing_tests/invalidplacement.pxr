Void main() {
  Pix test = new Pix();
  Placement placed = new Placement(text,200,300,3,"hello");
}