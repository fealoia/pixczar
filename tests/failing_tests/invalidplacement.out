Fatal error: exception Failure("undeclared identifier text")
